lambda/create/main.go

```diff
@@ -50,11 +53,11 @@ type Lambda struct {
 func (l *Lambda) HandleEvent(ctx context.Context, req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
 	_, err := l.verifier.VerifyHeader(req)
 	if err != nil {
-		l.logger.Print("Unable to verify JWT from header")
+		l.logger.Info("Unable to verify JWT from header")
```

Comment:
How come we would log this at info level rather than error?

Reply:
Error-level logs are for things which _shouldn't_ go wrong, since they trigger alarms. This is just informational, since LPA store doesn't particularly care about a unauthorised request.